scripts/spell-check-en-strings.rb

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'json'
+require 'open3'
+
+STRINGS_FILE = File.join('podcasts', 'en.lproj', 'Localizable.strings')
+KNOWN_WORDS = %w[
```

Comment:
Note that according to its man page, `aspell` is supposed to be able to take a `--personal={file}` parameter to provide a personal dictionary of known words. That being said, I tried to use it but it still reported the words I listed in my text file as typos in the output of the command. I probably missed something, and this could be worth looking a bit more into `aspell` to figure this out, instead of handling the logic ourselves in the ruby script? 🤷